gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -88,12 +88,13 @@ hook.Add("Think", "TTT2PlayerSprinting", function()
 
 		local timeElapsed = CurTime() - ply.sprintTS
 			
+		local modifier = {1} -- Multiple hooking support
 		if not ply.isSprinting then
-			local regenMult = hook.Run("TTT2StaminaRegen", ply)
-			ply.sprintProgress = math.min((ply.oldSprintProgress or 0) + timeElapsed * (regenMult or 1) * GetGlobalFloat("ttt2_sprint_stamina_regeneration"), 1)
+			local regenMult = hook.Run("TTT2StaminaRegen", ply, modifier)
```

Comment:
regenMult isn't needed anymore. Maybe we should remove this to prevent wrong hook usage